src/models/index.ts

```diff
@@ -1,2 +1,3 @@
 export {GroupBy} from './group-by';
 export {TimeSync} from './time-sync';
+export {ExhaustExportCsv} from './exhaust-export-csv';
```

Comment:
can be renamed to ExportCsv

Reply:
Fine by me, just wanted to make sure we don't to have some convention to differentiate regular plugins from exhaust plugins